scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -27,14 +32,35 @@ odo login -u developer -p developer
 # Check login user name for debugging purpose
 oc whoami
 
-# Integration tests
-make test-integration
-make test-integration-devfile
-make test-cmd-login-logout
-make test-cmd-project
-make test-operator-hub
+# import the odo-init-image for s390x arch
+if [ "${ARCH}" == "s390x" ]; then
+    export ODO_BOOTSTRAPPER_IMAGE=quay.io/zhengxiaomei123/odo4z:1.1.3-s390x
+fi
 
-# E2e tests
-make test-e2e-all
+if [ "${ARCH}" == "s390x" ]; then
+    # Integration tests
+    make test-generic
```

Comment:
Any reason for not to keep test in suite level like we did it for out testing ?

Reply:
How you are consuming ```scripts/openshiftci-presubmit-all-tests.sh``` ? in manual test ? or in CI ? Which CI infrastructure you are using ?

Reply:
> Any reason for not to keep test in suite level like we did it for out testing ?

Because there are some tests not work on s390x arch, so we skip it.

Reply:
> How you are consuming `scripts/openshiftci-presubmit-all-tests.sh` ? in manual test ? or in CI ? Which CI infrastructure you are using ?

In our local CI env which consumes `scripts/openshiftci-presubmit-all-tests.sh` manually.